roles/devscripts/tasks/300_post.yml

```diff
@@ -26,42 +26,39 @@
     - not cifmw_devscripts_ocp_online | bool
   ansible.builtin.import_tasks: set_cluster_fact.yml
 
-- name: Prepare for disk overlay configuration.
-  tags:
-    - devscripts_deploy
+- name: Handle OCP disk overlay
   when:
     - not cifmw_devscripts_ocp_comply | bool
-  ansible.builtin.include_tasks: 310_prepare_overlay.yml
-
-- name: Bringing cluster online.
-  when:
-    - not cifmw_use_reproducer | default(false) | bool
```

Comment:
ok, beware, this cifmw_use_reproducer was defaulting to "false". And toggled to "true" in the reproducer/tasks/ocp_layout.yml.

Here you're enabling those tasks bellow "by default", while it shouldn't be the case imho.

Since the condition is removed, you should probably remove those tasks instead.